examples/source_file_gpx_raster.html

```diff
@@ -59,29 +60,42 @@
             // Parse and Convert by iTowns
             itowns.Fetcher.xml('https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/ULTRA2009.gpx')
                 .then(function _(gpx) {
+                    var gpxStyle = new itowns.Style({
+                        stroke: {
+                            color: 'red',
+                        },
+                        point: {
+                            color: 'white',
+                            line: 'red',
+                        },
+                        text: {
+                            field: '{name}',
+                            transform: 'uppercase',
+                            font: ['Arial', 'sans-serif'],
+                            halo: {
+                                color: 'white',
```

Comment:
There's not halo in the gpx example?

Reply:
There is not anymore, because I can't merge all the styles unfortunately, so I have to choose a style among three: https://github.com/iTowns/itowns/pull/1303/files/9322d13d0f4a2fdf548bcc7704f684eec88d151e#diff-9229f99ccb02f6c4aa8d3d42de700a92R102
I am not sure how I can fix this problem yet.